Microsoft.Toolkit.Win32.sln

```diff
@@ -23,13 +23,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Toolkit.Wpf.UI.Xa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Toolkit.Forms.UI.Controls", "Microsoft.Toolkit.Forms.UI.Controls\Microsoft.Toolkit.Forms.UI.Controls.csproj", "{9B054ABA-B5B4-42F4-8E29-97DCCF79C726}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Toolkit.Sample.Forms.WebView", "Microsoft.Toolkit.Sample.Forms.WebView\Microsoft.Toolkit.Sample.Forms.WebView.csproj", "{D103E448-64B1-407C-B09E-7C61AF9F2740}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Toolkit.Sample.Forms.WebView", "Microsoft.Toolkit.Sample.Forms.WebView\Microsoft.Toolkit.Sample.Forms.WebView.csproj", "{D103E448-64B1-407C-B09E-7C61AF9F2740}"
```

Comment:
Why did the project GUIDs need to change?

Reply:
VS updated that for me, I don't know why it did that. Should I roll back?